cluster/common.sh

```diff
@@ -258,13 +258,14 @@ function load-or-gen-kube-bearertoken() {
 # current-context user does not exist or contain a server entry.
 function detect-master-from-kubeconfig() {
   export KUBECONFIG=${KUBECONFIG:-$DEFAULT_KUBECONFIG}
+  export KUBECTL_PATH=${KUBECTL_PATH:-${KUBE_ROOT}/cluster/kubectl.sh}
 
-  local cc=$("${KUBE_ROOT}/cluster/kubectl.sh" config view -o jsonpath="{.current-context}")
```

Comment:
This pattern exists all throughout `cluster.sh`, so changing this one place is fairly limited. Please expand this PR if you want to do this.